packages/core/src/app/payment/paymentMethod/HostedCreditCardFieldsetValues.ts

```diff
@@ -12,15 +10,6 @@ export default interface HostedCreditCardFieldsetValues {
     };
 }
 
-export interface HostedCreditCardValidationValues extends CardInstrumentFieldsetValues {
```

Comment:
Unused interface.
Currently using it in `packages/hosted-credit-card-integration/src/types/HostedCreditCardFiledsetValues.ts`.